ecs_pod_creator.go

```diff
@@ -867,6 +882,61 @@ func (s *SecretOptions) hash() string {
 	return h.Sum()
 }
 
+// LogConfiguration represents the configuration for a container's logging.
+type LogConfiguration struct {
+	// LogDriver is the logging driver to use.
+	LogDriver *string
+	// Options are the logging driver options.
+	Options map[string]*string
+}
+
+// NewLogConfiguration returns a new uninitialized log configuration.
+func NewLogConfiguration() *LogConfiguration {
+	return &LogConfiguration{}
+}
+
+// SetLogDriver sets the logging driver to use.
+func (c *LogConfiguration) SetLogDriver(logDriver string) *LogConfiguration {
+	c.LogDriver = &logDriver
+	return c
+}
+
+// SetOptions sets the logging driver options.
+func (c *LogConfiguration) SetOptions(options map[string]*string) *LogConfiguration {
```

Comment:
Same nit as above about variable length. Maybe `opts` or `o` is good enough for such a short function.